### PR TITLE
Update Job History

### PR DESCRIPTION
**Pull request by KimangKhenng**
- Added job in about-us page

### CODE REVIEW

File: content/km/about-us.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ biography: [
 Self-driving Cars"}
 ]
 works: [
+{ "date": "2024-04-01T05:28:04+0000", "to": "បច្ចុប្បន្ន", "note": "Instructor, CADT, Phnom Penh, Cambodia" },
```
Comment (tfd-ed) [inline content/km/about-us.md:15]: Thanks for your change.
I'd to make suggestion.
Please official title in Khmer language.

**Review by tfd-ed — CHANGES_REQUESTED** [review]
@KimangKhenng 
Please check my comment on the code.

File: content/km/about-us.md (hunk 1 above)
Comment (KimangKhenng) [reply content/km/about-us.md:15]: How should I put in title in Khmer?

Comment (tfd-ed) [reply content/km/about-us.md:15]: @KimangKhenng , Maybe you should use "គ្រូបង្រៀន"

Comment (KimangKhenng) [reply content/km/about-us.md:15]: Got it, I will change and update my PR

**Review by tfd-ed — APPROVED** [review]
LGTM

**Pull request merged**